packages/expo-web-browser/src/ExpoWebBrowser.web.ts

```diff
@@ -1,5 +1,15 @@
+import { BrowserResult, OpenBrowserParams } from './WebBrowser.types';
+
 export default {
   get name() {
     return 'ExpoWebBrowser';
   },
+  async openBrowserAsync(
+    url: string,
+    browserParams: OpenBrowserParams = {}
+  ): Promise<BrowserResult> {
+    const { target = ' ', features, replace } = browserParams;
+    window.open(url, target, features, replace);
```

Comment:
`replace` is not part of the spec: https://html.spec.whatwg.org/multipage/window-object.html#dom-open

Reply:
yas it is (maybe): https://www.w3schools.com/jsref/met_win_open.asp

Reply:
w3school is not the spec, there is location.replace for that: https://html.spec.whatwg.org/multipage/history.html#dom-location-replace

Reply:
I just copied over the TS definition so it surfaced all available props. Will remove.